cmd/release-controller/main.go

```diff
@@ -106,6 +108,8 @@ func main() {
 	var ignored string
 	flag.StringVar(&ignored, "to", ignored, "REMOVED: The image stream in the release namespace to push releases to.")
 	flag.StringVar(&opt.JobNamespace, "job-namespace", opt.JobNamespace, "The namespace to execute jobs and hold temporary objects.")
+	flag.StringVar(&opt.ProwJobCluster, "prow-job-cluster", opt.ProwJobCluster, "The cluster to configure on all created ProwJobs.")
+	flag.StringVar(&opt.ProwJobKubeconfig, "prow-job-kubeconfig", opt.ProwJobKubeconfig, "The kubeconfig to use for interacting with ProwJobs. Defaults to the main kubeconfig if unset.")
```

Comment:
How do we choose a context from this kubeconfig?

Reply:
We don't, its expected to only have one

Reply:
(If there is more than one, the code will load the `current_context`)

Reply:
In other places we do a more explicit mapping/identification of contexts. I'd love the kubeconfig for this to look as identical as possible to that for prow controlplane controllers, so as long as we're roughly in the same shape here I am OK

Reply:
We also dont do anything the like for the main kubeconfig. Doing so is only required if there is a need to pass more than one config in one kubeconfig IMHO